lib/App.js

```diff
@@ -1820,21 +1817,17 @@ class App {
   }
 
   async createFlow() {
-    let flowJson = await this.createFlowJson();
-
-    if( !flowJson ) throw new Error('Could not create valid Flow');
```

Comment:
Why not throw this error?

Reply:
cause the only way this returns undefined is if the user chose to skip, so we don't really need to show an error in that case